rust/core-lib/src/view.rs

```diff
@@ -586,7 +585,7 @@ impl View {
     ///
     /// Return value is a location of a point that should be scrolled into view.
```

Comment:
Need to update comment, something like "Requests the point that should be scrolled into view."